GsaGH/Parameters/4_Analysis/IGsaDesignTask.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GsaAPI;
+using GsaGH.Helpers;
+
+namespace GsaGH.Parameters {
+  /// <summary>
```

Comment:
do we need that comment here?

Reply:
Yes it's being reflected in the docs generator